cbmc/proofs/IotMqtt_SubscribeAsync/IotMqtt_SubscribeAsync_harness.c

```diff
@@ -0,0 +1,107 @@
+#include "iot_config.h"
+#include "private/iot_mqtt_internal.h"
+
+#include <stdlib.h>
+
+#include "mqtt_state.h"
+
+/**
+ * We abstract all functions related to concurrency and assume they are correct.
+ * Thus, we add these stub to increase coverage.
+ */
+IotTaskPoolError_t IotTaskPool_CreateJob( IotTaskPoolRoutine_t userCallback,
+                                          void * pUserContext,
+                                          IotTaskPoolJobStorage_t * const pJobStorage,
+                                          IotTaskPoolJob_t * const pJob )
+{
+  if (userCallback == NULL || pJobStorage == NULL || pJob == NULL)
```

Comment:
In my current branch, I had to replace this if statement with three instances of assert(userCallback != NULL).  The reason is that this function is always called correctly, so the return IOT_TASKPOOL_BAD_PARAMETER statement is never executed, so my own modeling code is contributing to a decrease in coverage.

Reply:
Done!